### DISCUSSION TITLE
How to add new line in contextual information

### DISCUSSION THREAD

**Discussion by simkimsia** [commitizen-tools/commitizen#859] [Q&A] [answered]
<img width="883" alt="image" src="https://github.com/commitizen-tools/commitizen/assets/245021/bcd0332c-8c3a-4343-a48e-3ba94f0f56a4">

I have tried shift+enter and alt+enter on a macOS. None work. What can i do for next line for the question `Provide additional contextual information about the code changes:`

**Comment by woile** [comment, marked as answer]
As of now new lines are separated by a pipe: `|`

`This goes in one line | this would go to the next one`

There are some ideas to move towards something better. 
Apologies for the confusion

**Reply by simkimsia** [reply]
Separate question

Using pipe to delimit as new lines

Is this a limitation inherited from questionary?

**Reply by woile** [reply]
It was, but I think it now supports a different approach, we haven't investigated further
